internal/authtoken/repository.go

```diff
@@ -196,11 +201,13 @@ func (r *Repository) ValidateToken(ctx context.Context, id, token string, opt ..
 	// If the token is too old or stale invalidate it and return nothing.
 	exp, err := ptypes.Timestamp(retAT.GetExpirationTime().GetTimestamp())
 	if err != nil {
-		return nil, fmt.Errorf("validate token: expiration time : %w", err)
+		// hmm
+		return nil, errors.Wrap(err, op, errors.WithMsg("expiration time"))
```

Comment:
Yeah, we will likely need to make a new code for this...perhaps just `InvalidTimeStamp` @jimlambrt thoughts?

Reply:
I think, that sounds reasonable.   It's a common error